### PR TITLE
Fix: APP-2014 - Deposit & Withdraw of non 18 decimal tokens

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- Fixes the issue that happens when depositing & withdrawing non-18 decimal tokens
- Pulls in the token price and image on Proposal details page

Task: [APP-2014](https://aragonassociation.atlassian.net/browse/APP-2014)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2014]: https://aragonassociation.atlassian.net/browse/APP-2014?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/containers/configureWithdraw/index.tsx [hunk 1]
```diff
@@ -115,25 +115,27 @@ const ConfigureWithdrawForm: React.FC<ConfigureWithdrawFormProps> = ({
...
-          setValue(`actions.${actionIndex}.tokenSymbol`, symbol);
+        const [balance, apiData, chainData] = await allTokenInfoPromise;
+        if (apiData) {
+          setValue(`actions.${actionIndex}.tokenName`, apiData.name);
+          setValue(`actions.${actionIndex}.tokenSymbol`, apiData.symbol);
+          setValue(`actions.${actionIndex}.tokenImgUrl`, apiData.imgUrl);
+          setValue(`actions.${actionIndex}.tokenPrice`, apiData.price);
         }
+
+        if (!apiData) {
```
Comment (sepehr2github) [inline packages/web-app/src/containers/configureWithdraw/index.tsx]: Shouldn't be if !apiData && chainData?

**Review by sepehr2github — COMMENTED** [review]
Thanks, looks good to me, I just left a simple suggestion, please let me know what you think

Quick thing, Have you disabled the cache because the deposit didn't come for me after submitting the transaction

**Comment by Fabricevladimir** [comment]
@sepehr2github 
> Quick thing, Have you disabled the cache because the deposit didn't come for me after submitting the transaction

Cache is virtually untouched. Also, I can't reproduce. The deposit came in immediately

**Review by sepehr2github — APPROVED** [review]
thanks for the changes

**Pull request merged**